rbi/annotations/actionview.rbi

```diff
@@ -3,7 +3,7 @@
 module ActionView
   TemplateError = T.type_alias { Template::Error }
 
-  class MissingTemplate < ActionViewError
+  class MissingTemplate < ActionView::ActionViewError
```

Comment:
The problem here is that the class is not flattened properly and thus the superclass name is not fully qualified. We could be smarter but I think it's a good thing to enforce fully qualified names